src/main/java/com/spring/core/userinterface/UserInterface.java

```diff
@@ -0,0 +1,59 @@
+package com.spring.core.userinterface;
+
+import com.spring.core.userinterface.impl.UserMenuChoice;
+
+/**
+ * this interface provides methods to manage products basket from console
```

Comment:
Interface shouldn't be depends on details like "from console"
It's an abstraction